core-web/libs/dot-primeng-theme-styles/src/scss/dotcms-theme/components/_button.scss

```diff
@@ -318,9 +318,10 @@
 }
 
 .p-button.p-button-link {
-    color: transparent;
+    color: $blue-link;
```

Comment:
This should definitely be primary color.

Reply:
And looks better. Changed.